ehrenamt-justiz-eai/README.md

```
@@ -46,18 +46,30 @@
+| api.auth.users.username | User for authentication on this EAI                                                                                                                                                                  |  |
+| api.auth.users.password | Password for authentication on this EAI                                                                                                                                                              |  |
 
 ## SoapUI
 
 The EAI can be tested with SoapUI. The xml project to be imported into SoapUI can be found here:
 
 [ehrenamt-justiz-soapui-project.xml](https://github.com/it-at-m/ehrenamt-justiz/blob/main/ehrenamt-justiz-eai/src/test/resources/ehrenamt-justiz-soapui-project.xml)
 
+Example of an EAI call in docker desktop with the parameters first name, last name and date of birth:
+
+<p>
```

markdownlint-cli2 (0.17.2)

Inline HTML
Element: p

(MD033, no-inline-html)